src/components/common/UserDetails.tsx

```
@@ -56,13 +57,15 @@ function chunkArray<T>(arr: T[], size: number): T[][] {
 }
 
 const UserDetails: React.FC<UserDetailsProps> = ({ userData }) => {
+	const { t } = useTranslation();
+	
```

ESLint

[error] Delete `↹`

(prettier/prettier)